--- modulemd-2.15.0/meson.build.orig	2023-05-20 11:03:06.755527932 +0200
+++ modulemd-2.15.0/meson.build	2023-05-20 11:05:27.564765103 +0200
@@ -69,21 +69,7 @@ test = find_program('test')
 with_docs = get_option('with_docs')
 if with_docs
   gtkdoc = dependency('gtk-doc')
-  glib_docpath = join_paths(glib_prefix, 'share', 'gtk-doc', 'html')
-
-  glib_index_path = join_paths(glib_docpath, 'glib/index.html')
-  ret = run_command ([test, '-e', glib_index_path],
-    check: false)
-  if ret.returncode() != 0
-    error('Missing documentation for GLib: @0@'.format(glib_index_path))
-  endif
-
-  gobject_index_path = join_paths(glib_docpath, 'gobject/index.html')
-  ret = run_command ([test, '-e', gobject_index_path],
-    check: false)
-  if ret.returncode() != 0
-    error('Missing documentation for GObject: @0@'.format(gobject_index_path))
-  endif
+  glib_docpath = get_option('glib_docpath')
 endif
 
 # Keep with_manpages option a tristate feature for backward compatibility.
--- modulemd-2.12.0/meson_options.txt~	2021-01-15 03:08:36.000000000 +0100
+++ modulemd-2.12.0/meson_options.txt	2021-03-13 21:09:46.821667993 +0100
@@ -32,6 +32,8 @@
 option('with_docs', type : 'boolean', value : true,
        description : 'Build API documentation.')
 
+option('glib_docpath', type : 'string')
+
 option('with_manpages', type : 'feature', value : 'auto',
        description : 'Build manual pages for included executables.')
 
